user_tools/src/spark_rapids_pytools/rapids/profiling.py

```diff
@@ -247,7 +247,8 @@ def _process_output(self):
         self.__generate_report_with_recommendations()
 
     def _init_rapids_arg_list(self) -> List[str]:
-        return self._create_autotuner_rapids_args()
+        platform_arg = ['--platform', self.ctxt.platform.get_platform_name().replace('_', '-')]
+        return [platform_arg] + self._create_autotuner_rapids_args()
```

Comment:
Thanks @cindyyuanjiang. Can we use the parent method's definition here to avoid duplication?
```suggestion
        return super()._init_rapids_arg_list() + self._create_autotuner_rapids_args()
```